### PR TITLE
fly/cmd: add new tables

### PR DESCRIPTION
**Pull request by panoel**

### CODE REVIEW

File: fly/cmd/heartbeats/main.go [hunk 1]
```diff
@@ -183,36 +324,75 @@ func main() {
...
 	// Note: without this, the whole program hangs on observation requests
 	go func() {
 		for {
 			select {
 			case <-rootCtx.Done():
 				return
 			case <-obsvReqC:
+				// There is no guardian address in the observation request
+				// gossipCounter[idx][GSM_signedObservationRequest]++
+				gossipCounter[totalsRow][GSM_signedObservationRequest]++
```
Comment (bruce-riley) [inline fly/cmd/heartbeats/main.go:370]: Should this be inside the lock?

File: fly/cmd/heartbeats/main.go [hunk 2]
```diff
@@ -341,6 +556,54 @@ func resetTerm(clear bool) {
...
+	needToRender := false
+	now := time.Now()
+	if now.Sub(lastTime) > time.Minute {
+		lastTime = now
+		for idx, r := range currentObsvData {
+			currentObsvTable[idx] = r
+		}
+		for i := 0; i < numGuardians; i++ {
+			obsvRateRows[i].obsvCount = currentObsvTable[uint(i)]
+			pct := currentObsvTable[uint(i)] * 100 / currentObsvTable[numGuardians]
```
Comment (bruce-riley) [inline fly/cmd/heartbeats/main.go:590]: Should `numGuardians` be `totalsRow`? Same with line 603.

**Review by bruce-riley — APPROVED** [review]

File: fly/cmd/heartbeats/main.go (hunk 2 above)
Comment (panoel) [reply fly/cmd/heartbeats/main.go:590]: Fixed.

File: fly/cmd/heartbeats/main.go [hunk 3]
```diff
@@ -121,7 +207,42 @@ func main() {
...
+	const totalsRow = numGuardians
+	if len(gs.Keys) != numGuardians {
+		logger.Error("Invalid number of guardians.", zap.Int("found", len(gs.Keys)), zap.Uint32("expected", numGuardians))
+		return
+	}
+
+	var gossipLock sync.Mutex
+	// The extra row is for the totals
+	var gossipCounter = [numGuardians + 1][GSM_maxTypeVal]int{
+		{0, 0, 0, 0, 0, 0, 0},
```
[the remark this answers is not in the document]
Comment (panoel) [reply fly/cmd/heartbeats/main.go:224]: It looks like I do NOT have to do this.  Removed.

**Review by evan-gray — APPROVED** [review]

**Pull request merged**